Gemfile

```diff
@@ -1,5 +1,7 @@
 source 'https://rubygems.org'
 
+ruby '2.3.1'
```

Comment:
Circle wants 2.3.1, while heroku wants 2.3.3 (due to an earlier choice).  Docker is on 2.3.1 isn't it?